src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String str;
+    private int variable;
```

Comment:
What does `variable` mean?

Reply:
renamed the "variable" field to "year" for easy reading of the fields.